test/test_torch.py

```diff
@@ -10560,6 +10560,16 @@ def test_linspace(self, device):
         y = torch.linspace(0, 3, 4, out=x.narrow(1, 1, 2))
         self.assertEqual(x, torch.tensor(((0, 0, 1), (0, 2, 3)), device=device), 0)
 
+    def test_linspace_integral(self, device):
```

Comment:
can you split this into another PR (or just remove this) so the PRs are basically the same on master and v1.4.0?  Otherwise it becomes a bit harder to track what is what.